File: Tutorials/WinForms/DevExpress.MVVM/CS/Views/EditOrderView.cs
using DevExpress.XtraEditors;
using WinFormsApplication.ViewModels;

namespace WinFormsApplication.Forms {
    public partial class EditOrderView : XtraForm {
        public EditOrderView() {
            InitializeComponent();
            if(!mvvmContext1.IsDesignMode)
                InitializeBindings();
        }
        void InitializeBindings() {
            var fluentApi = mvvmContext1.OfType<EditOrderViewModel>();
            fluentApi.SetBinding(OrderBindingSource, bs => bs.DataSource, x => x.Order);
            fluentApi.SetBinding(CustomersBindingSource, bs => bs.DataSource, x => x.Customers);
            //
            fluentApi.BindCommand(btnSave, x => x.Save());
            fluentApi.BindCommand(btnReload, x => x.Reload());
        }
    }
}

File: Tutorials/WinForms/DevExpress.MVVM/VB/Views/EditOrderView.vb
Imports DevExpress.XtraEditors
Imports WinFormsApplication.ViewModels

Namespace WinFormsApplication.Forms
	Partial Public Class EditOrderView
		Inherits XtraForm

		Public Sub New()
			InitializeComponent()
			If Not mvvmContext1.IsDesignMode Then
				InitializeBindings()
			End If
		End Sub
		Private Sub InitializeBindings()
			Dim fluentApi = mvvmContext1.OfType(Of EditOrderViewModel)()
			fluentApi.SetBinding(OrderBindingSource, Function(bs) bs.DataSource, Function(x) x.Order)
			fluentApi.SetBinding(CustomersBindingSource, Function(bs) bs.DataSource, Function(x) x.Customers)
			'
			fluentApi.BindCommand(btnSave, Sub(x) x.Save())
			fluentApi.BindCommand(btnReload, Sub(x) x.Reload())
		End Sub
	End Class
End Namespace
